site/public/js/configuration.js

```diff
@@ -6,8 +6,14 @@ $(document).ready(() => {
         const formData = new FormData();
         formData.append('csrf_token', csrfToken);
         let entry;
+        let default_section;
         if (this.type === 'checkbox') {
             entry = $(elem).is(':checked');
+            if (this.id === 'self-registration') {
+                console.log('test');
```

Comment:
I assume this line wasn't intentional, and should be reverted?

Reply:
It was a test, and an unintentional addition, and has been reverted.